client/src/components/Submissions/Submissions.jsx

```diff
@@ -16,9 +16,7 @@ const SubmissionsUpload = () => {
   };
 
   const { data: allSystems } = useSystems();
-  const submissionSystem = allSystems.find((s) => {
-    return s.system && s.system.startsWith('apcd.submissions');
-  });
+  const submissionSystem = allSystems.find((s) => s.site === 'apcd');
```

Comment:
maybe make the site it's looking for `apcd.submissions`, just in case more systems come later with the same site name